api/packages.go

```diff
@@ -32,6 +32,11 @@ func (resp *ValidationResponse) IsValid() bool {
 	return (len(resp.InstallationErrors) == 0) && (len(resp.ParseErrors) == 0)
 }
 
+// InstalledPackage contain the details of an installed package
+type InstalledPackage struct {
```

Comment:
Please add more fields to the struct. This is part of the API package and I would think that users of the API package would like more info than an ID only.

Reply:
I don't agree. Think this is a case of "You ain't gonna need it."
For now we don't need anything else and until we have a way to pass a selection set, this is appropriate.

Reply:
Alright. I see also that `ID` in this case is `foo/bar@1.2.3` not a random string.

Reply:
I added a few more fields just because I didn't see this comment before I was done.